project-repo/templates/common/.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,95 @@
+        sensitive information, use [Launchpad](https://bugs.launchpad.net/ubuntu/+source/PROJECT_TODO/+filebug) instead.
+        On Ubuntu machines, you can use `ubuntu-bug PROJECT_TODO` to collect relevant information.
+
+
+        Thanks for taking the time to report an issue and help improve PROJECT_TODO! Please fill out the form below as
+        best as you can so that we can help you.
+
+        Your additional work here is greatly appreciated and will help us respond as quickly as possible. For general
+        support or usage questions, use [Ubuntu Discourse](https://discourse.ubuntu.com/c/desktop/8).
+
+        By submitting an Issue to this repository, you agree to the terms within the
+        [Ubuntu Code of Conduct](https://ubuntu.com/community/code-of-conduct).
+
+
+        FIXME: snap?
+  - type: checkboxes
+    attributes:
+      label: Is there an existing request for this feature?
+      options:
+        - label: I have searched the existing issues and found none that matched mine
+          required: true
+  - type: textarea
+    attributes:
+      label: Describe the feature
+      description: >
+        A clear and concise description of what the problem is. Ex.: \"I'm always frustrated when ...\"
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Describe the ideal solution
+      description: >
+        A clear and concise description of what you want to happen.
+  - type: textarea
+    attributes:
+      label: Alternatives and current workarounds
+      description: >
+        A clear and concise description of any alternatives you've considered or any workarounds that are currently in
+        place.
+  - type: textarea
+    attributes:
+      label: "Ubuntu users: System information and logs"
+      description: >
+        Ubuntu users can run `ubuntu-bug PROJECT_TODO --save=/tmp/report.txt` and drag the file below.
+        It will contain any useful information.
+  - type: textarea
+    attributes:
+      label: "Non Ubuntu users: System information and logs"
+      description: |
+        For users of other distributions than ubuntu, provide details about the environment you experienced the issue in:
+      value: |
+        ### Environment
+        * PROJECT_TODO version: please run TODO
+        * Distribution: (**NAME** in `/etc/os-release`)
+        * Distribution version: (**VERSION_ID** on `/etc/os-release`):
+
+        ### Log files
+        Please redact/remove sensitive information:
+        ```raw
+        TODO: how to collect them
+        ```
+
+        ### Application settings
+        Please redact/remove sensitive information:
+        ```raw
+        TODO: how to collect them
+        ```
+
+        FIXME: snap?
+  - type: textarea
+    attributes:
+      label: Relevant information
+      description: >
+        Please look at our
+        [PROJECT_TODO Troubleshooting guide](../#troubleshooting)
```

Comment:
Have you tested this with a real bug? I think the github UI renders the page, and so, has the readme attached and the link then may work, but just double checking :)

Reply:
I tested it, yes. If we use the `../` it points to the homepage of the repo (you can check on the repo that I've shared with you) and then the `#troubleshooting` directs you to the `##Troubleshooting` item